File: dfas-monitor-common/src/main/java/com/win/dfas/monitor/common/vo/MicroServiceMachineRepVO.java
package com.win.dfas.monitor.common.vo;

import com.win.dfas.common.vo.BaseRepVO;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class MicroServiceMachineRepVO extends BaseRepVO {

    /**
     * 实例ID
     */
    private String instanceId;

    /**
     * IP地址
     */
    private String ipAddr;

    /**
     * 端口
     */
    private String port;

    /**
     * 状态
     */
    private String state="DOWN";

    /**
     * 负载
     */
    private String balance="-";

    /**
     * jvm
     */
    private String jvm="-";

    /**
     * 告警数
     */
    private String warn="-";

    /**
     * 错误数
     */
    private String error="-";

}

File: dfas-monitor-web/src/main/frontend/src/page/microService/vo/MicroServiceMachineRepVO.ts
/****************************************************
 * 创建人：  @author wangyaoheng
 * 创建时间: 2019-07-10 11:35:35
 * 文件名称: MicroServiceInfoVO.ts
 * 文件描述: @Description: 微服务 返回VO
 *
 * All rights Reserved, Designed By 投资交易团队
 * @Copyright:2016-2019
 *
 ********************************************************/

import {BaseRepVO} from "../../common/vo/BaseVO";

/**
 * 类名称：MicroServiceInfoRepVO
 * 类描述：交易对手表 返回VO
 * 创建人：@author wangyaoheng
 * 创建时间：2019-07-10 11:35:35
 */
export class MicroServiceMachineRepVO extends BaseRepVO {

  /**
   * IP地址
   */
  public ipAddr: string;

  /**
   * 状态
   */
  public state: string;

  /**
   * 负载
   */
  public balance: string;

  /**
   * jvm
   */
  public jvm: string;

  /**
   * 告警数
   */
  public warn:string;

  /**
   * 错误数
   */
  public error:string;
}
